src/redprl/refiner.fun

```diff
@@ -17,7 +17,7 @@ struct
 
   structure Hyp =
   struct
-    fun Project z alpha jdg =
+    fun Project z _ jdg =
```

Comment:
To brainstorm the ideas, I have tried `ignoreAlpha @@ fn jdg =>...` but SML wants parentheses around the `fn` expression, which in my opinion is ugly. An also ugly compromise is to define `EqType'` and then `fun EqType jdg = ignoreAlpha EqType' jdg`.